MonoGame.Framework/Content/ContentReader.cs

```diff
@@ -118,7 +118,7 @@ public T ReadExternalReference<T>()
 
             if (!String.IsNullOrEmpty(externalReference))
             {
-                return contentManager.Load<T>(FileHelpers.ResolveRelativePath(assetName, externalReference));
+                return contentManager.Load<T>(MonoGame.Utilities.FileHelpers.ResolveRelativePath(assetName, externalReference));
```

Comment:
Use a `using Namespace`

Reply:
Ok. thanks i'm go refer the name space. Thanks for reply.